backend/qemu.pm

```diff
@@ -874,6 +874,13 @@ sub start_qemu {
         for (my $i = 0; $i < $num_networks; $i++) {
             if ($vars->{NICTYPE} eq "user") {
                 my $nictype_user_options = $vars->{NICTYPE_USER_OPTIONS} ? ',' . $vars->{NICTYPE_USER_OPTIONS} : '';
+                if ($vars->{QEMU_ENABLE_SMBD}) {
+                    my $smbd_bin = find_bin('/usr/sbin/', 'smbd');
```

Comment:
Any particular reason why you use `find_bin` from the os-autoinst module osutils? I don't think this should be necesary here. AFAIU `find_bin` in particular is useful when we want to lookup binaries in multiple alternative paths but here AFAICS you can rely on "smbd" from PATH, isn't it?